Лабораторная работа 1/task_3.py

```diff
@@ -0,0 +1,10 @@
+list_players = ["Маша", "Петя", "Саша", "Оля", "Кирилл", "Коля"]
+
+# индекс середины
+middle_index = len(list_players) // 2
+
+first_team = list_players[:3]
```

Comment:
В данной строке вы производите слайсирование по "жестко" введенному индексу. Такое решение не является верным, так как если список имен измениться, то решение выполнит деление команд не корректно. В строке ранее вы находили индекс середины, который и надо использовать по ходу решения далее. 
```suggestion
first_team = list_players[:middle_index]
```